app/services/albums.js

```diff
@@ -36,6 +36,13 @@ const albumPurchased = (albumId, userId) =>
       throw error.databaseError(err.message);
     });
 
+exports.getAlbumsListByIdUser = userId =>
+  Album.findAll({ where: { userId } })
+    .then(result => result)
```

Comment:
I'm not going to say anything here, you know... 😄😄😄😄😄😄

Reply:
perfect!